mocks/wsHandlerStub.go

```diff
@@ -0,0 +1,17 @@
+package mocks
+
+import "net/http"
+
+// WSHandlerStub implements WSHandler interface
+type WSHandlerStub struct {
+	ServeHTTPCalled func(w http.ResponseWriter, r *http.Request)
+}
+
+// ServeHTTP -
+func (whs *WSHandlerStub) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+}
```

Comment:
forgot to call `ServeHTTPCalled` on this function

Reply:
right :+1: thanks